src/classrooms/classrooms.service.ts

```diff
@@ -83,4 +83,14 @@ export class ClassroomsService {
     // For now, throw ForbiddenException if the user is not the teacher
     return classroom;
   }
+
+  async deleteClassroom(id: number, userId: number): Promise<boolean> {
+    const classroom = await this.getClassroomById(id, userId);
+    if (!classroom) {
+      return false;
+    }
+
+    await this.em.removeAndFlush(classroom);
```

Comment:
@ishmam-iqbal-sazim Shouldn't we do the remove and flush stuff inside the repository?

Reply:
This was my suggestion since this is a built in method provided by the repository and the name is self explanatory. An extra abstraction method such as `deleteOne` may be unnecessary for such a simple use case.